java/src/jmri/jmrit/Bundle.properties

```diff
@@ -246,3 +246,12 @@ MenuCurrentMeters  = Current meters
 MenuMeterSettings  = Settings
 MenuMeterIntegerDigits = {0} integer digits
 MenuMeterDecimalDigits = {0} decimal digits
+
+# moved from SpeedProfileBundle.properties
+mm	        = millimeters/sec
```

Comment:
I think it would be good to rename this to mmps or something similar. If we later need the unit `millimeters`, it's bad that `mm` is used for `millimeters/sec`.

Reply:
I will do it.

Reply:
Thanks.

Reply:
Done